### PR TITLE
Use gjspack

### PR DESCRIPTION
**Pull request by sonnyp**

### CODE REVIEW

File: src/meson.build [hunk 1]
```diff
@@ -58,5 +54,24 @@ configure_file(
   install_dir: get_option('bindir')
 )
 
-install_data('workbench-api.vala', install_dir: datadir)
+install_data('workbench-api.vala', install_dir: pkgdatadir)
```
Comment (sonnyp) [inline src/meson.build:57]: @lw64 FYI

Comment (lw64) [reply src/meson.build:57]: the command to run `valac` needs to be adjusted, if they are different paths.

Comment (lw64) [reply src/meson.build:57]: here: https://github.com/sonnyp/Workbench/blob/main/src/Compiler.js#L16

File: src/Compiler.js [hunk 2]
```diff
@@ -13,7 +13,7 @@ export default function Compiler(data_dir) {
     GLib.build_filenamev([data_dir, "libworkbenchcode.so"])
   );
   const api_file = Gio.File.new_for_path(
-    GLib.build_filenamev(["/app/share", "workbench-api.vala"])
+    GLib.build_filenamev([pkg.pkgdatadir, "workbench-api.vala"])
```
Comment (lw64) [inline src/Compiler.js:16]: perfect! didnt saw this :)

**Pull request merged**